server/controller/partyController.js

```diff
@@ -75,20 +75,21 @@ class PartyController{
 
         })
 
-        return res.status(400).send({
-            status: 400,
+        return res.status(404).send({
+            status: 404,
             error: "Party not found"
         })
         
     }
 
     editAParty(req, res){
-        const id = parseInt(req.params.id);
-        const name = req.params.name;
+        const param_id = parseInt(req.params.id);
```

Comment:
You are still getting the value for the URL params which is not suppose to be so., Please get your details from the JSON body.

Reply:
Ok